data-raw/make_cluster_vcov.R

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Need to add this folder to `.Rbuildignore`, e.g., https://github.com/hadley/babynames/blob/master/.Rbuildignore

See also http://r-pkgs.had.co.nz/data.html